spec/decorators/odk/form_decorator_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+# Form decorator spec
+describe Odk::FormDecorator, :odk, :reset_factory_sequences do
+  include_context "odk rendering"
+
+  describe "needs_manifest?" do
+    let(:decorated_form) { decorate(form) }
```

Comment:
Perfect!

Reply:
Now here's another fun thing!

This is a great place to use `subject` and the one-liner syntax for expect.

Up here, put `subject(:needs_manifest) { decorated_form.needs_manifest? }`

Then below, instead of the current `it` block, try `it { is_expected.to be false }`

Sweet!